code/game/objects/items/storage/boxes.dm

```diff
@@ -432,3 +432,61 @@
 	if(!isopened)
 		isopened = 1
 		icon_state = "mealpackopened"
+
+//Fillable Ammo Box
+/obj/item/storage/box/ammo
+	name = "\improper Ammo Box"
+	desc = "A large ammo box that can be filled with almost any magazine type. Compact and deployable for moving ammo to the front lines."
+	icon_state = "ammobox"
+	max_w_class = 4
+	storage_slots = 30
+	max_storage_space = 60	//SMG and pistol sized (tiny and small) mags can fit all 30 slots, normal (LMG and AR) fit 20
```

Comment:
so is this for mixed magazine types?  if so searching through such a huge box that may contain empty/partial magazines would be hell for players

Reply:
It would take a real dick bag and effort to fill a box with partially empty mags. Besides you can see if the mag is empty due to the sprite. Or just examine it. Non-issue.